stl/inc/utility

```diff
@@ -26,6 +26,36 @@ _STL_DISABLE_CLANG_WARNINGS
 #pragma push_macro("new")
 #undef new
 
+#if defined(_CRTBLD) && defined(CRTDLL2)

```

Comment:
By any chance could I convince you to push this further up into `<type_traits>` so we can also modify

https://github.com/microsoft/STL/blob/314f65f831f827dbfb05ebcc0f08c496f5ddf64f/stl/inc/concepts#L150-L157

similarly?

Reply:
Sure, great catch! File this one under "STL no longer remembers everything that's in this library" :joy_cat:

Reply:
Meh, it's my fault for not truly using `std::swap_ranges` to implement the array behavior of `ranges::swap`.